warpgate-sso/src/error.rs

```diff
@@ -24,6 +24,8 @@ pub enum SsoError {
     Io(#[from] std::io::Error),
     #[error("JWT error: {0}")]
     Jwt(#[from] jsonwebtoken::errors::Error),
+    #[error("signature verification: {0}")]
+    SignatureVerification(#[from] SignatureVerificationError),
```

Comment:
What's the purpose of this new error?

(I'm asking to learn, not asking because I'm thinking is deflect or error :))

Reply:
It's to allow auto-converting the error from `exchange_code` into `SsoError` via `?`

Reply:
Yes 5 mins after I posted the comment (during shower, I always have the eureka moment in shower) I understand that by moving from .unwrap() to ?. The new error was here for the mapping

Thank you 

I really need to practice more rust, but isn't my daily language I need to find project.